src/clr-angular/data/datagrid/datagrid-row-detail.ts

```diff
@@ -21,18 +22,30 @@ import {Selection, SelectionType} from "./providers/selection";
 @Component({
     selector: "clr-dg-row-detail",
     template: `
-        <ng-container *ngIf="!replace">
-            <clr-dg-cell class="datagrid-fixed-column"
-                *ngIf="selection.selectionType === SELECTION_TYPE.Multi 
-                    || selection.selectionType === SELECTION_TYPE.Single"></clr-dg-cell>
-            <clr-dg-cell *ngIf="rowActionService.hasActionableRow" class="datagrid-fixed-column"></clr-dg-cell>
-            <clr-dg-cell class="datagrid-fixed-column"></clr-dg-cell>
-        </ng-container>
+        <!-- space for multiselection state -->
+        <div class="datagrid-cell datagrid-select datagrid-fixed-column"
```

Comment:
Because I moved the check/radio's and action overflow buttons out of the sticky container (due to the z-index issue) I need to have space for them in the details view for the use case when one or more is present and cells are not replaced. 

tbh, I copied these changes from the header though and we use three placeholder elements there. Either way the conditional logic should be the same for header, row and detail elements. I'll change this or the header to be the same. 

fyi - The UX recommendation is not to have all three in a row at the same time iir correctly so consumers should only have 2 of these elements at a time.

Reply:
checking the templates, I made them all individual elements with the correct logic for each element on the `*ngIf`